src/main/resources/xslt/preservica2solr.xsl

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xsl:transform xmlns:t="http://www.test.com/"
+               xmlns:m="http://www.loc.gov/mods/v3"
+               xmlns:mets="http://www.loc.gov/METS/"
+               xmlns:tei="http://www.tei-c.org/ns/1.0"
+               xmlns:f="http://www.w3.org/2005/xpath-functions"
+               xmlns:h="http://www.w3.org/1999/xhtml"
+               xmlns:xs="http://www.w3.org/2001/XMLSchema"
+               xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+               xmlns:my="urn:my"
+               xmlns:premis="http://www.loc.gov/premis/v3"
+               xmlns:mix="http://www.loc.gov/mix/v20"
+               xmlns:pbc="http://www.pbcore.org/PBCore/PBCoreNamespace.html"
+               xmlns:xip="http://example.com/"
+               version="3.0">
+
+  <xsl:output method="text" />
+  <!-- Declare all externally provided parameters here -->
+  <!-- Provided by the Java code that calls the Transformer. (Key,Value) pairs are given in a map -->
+  <xsl:param name="access_blokeret"/>
+  <xsl:param name="access_pligtafleveret"/>
+  <xsl:param name="access_ejermaerke"/>
+  <xsl:param name="access_note"/>
+  <xsl:param name="access_skabelsesaar"/>
+  <xsl:param name="access_ophavsperson_doedsaar"/>
+  <xsl:param name="access_searlige_visningsvilkaar"/>
+  <xsl:param name="access_materiale_type"/>
+  <xsl:param name="access_foto_aftale"/>
+  <xsl:param name="access_billede_aftale"/>
+  <xsl:param name="access_ophavsret_tekst"/>
```

Comment:
I don't think all of these will work for video?

Reply:
Correct. I will do some pruning and only keep what is currectly working